CleverPush/Source/CleverPushInstance.m

```diff
@@ -1216,6 +1225,7 @@ - (void)setSyncSubscriptionRequestData:(NSMutableURLRequest*)request {
                                     isNil(country), @"country",
                                     isNil(timezone), @"timezone",
                                     isNil(language), @"language",
+                                    [NSNumber numberWithBool:autoRequestNotificationPermission], @"hasNotificationPermission",
```

Comment:
Here we do not want to send the value of `autoRequest`. Instead we want to send the authorizationStatus - has user given the permission?